samples/msal-node-samples/client-credentials-distributed-cache/src/AxiosHelper.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import axios from "axios";
+
+class AxiosHelper {
+
+    /**
+     * Makes an Authorization "Bearer" request with the given accessToken to the given endpoint.
+     * @param endpoint
+     * @param accessToken
+     */
+    static async callEndpointWithToken(endpoint: string, accessToken: string | undefined): Promise<any> {
```

Comment:
Can this be `undefined`?  and should we check for it?

Reply:
this would work (u would get 401 if the endpoint needs a token), but I should probably make this argument optional instead -will update